Common/src/main/java/at/petrak/hexcasting/api/casting/eval/vm/CastingVM.kt

```diff
@@ -74,7 +74,7 @@ class CastingVM(var image: CastingImage, val env: CastingEnvironment) {
             lastResolutionType = image2.resolutionType
             try {
                 performSideEffects(image2.sideEffects)
-            } catch (e: Exception) {
+            } catch (e: RuntimeException) {
```

Comment:
This should stay as Exception. We want to catch all exceptions here, to prevent inadvertently-thrown exceptions from escaping the VM and crashing the entire server.